logging/logging_alerts/log-collector-alerts.adoc

```diff
@@ -0,0 +1,13 @@
+:_content-type: ASSEMBLY
+[id="log-collector-alerts"]
+include::_attributes/common-attributes.adoc[]
+= Log collector alerts
+:context: log-collector-alerts
+
+toc::[]
+
+Log collector alerts are enabled by default when you install the Cluster Logging Operator, as long as you have selected the option to *Enable operator recommended cluster monitoring on this namespace* when installing the Cluster Logging Operator. For more information about installing logging Operators, see xref:../../logging/cluster-logging-deploying#cluster-logging-deploy-console_cluster-logging-deploying[Installing the {logging-title} using the web console].
```

Comment:
Log collector alerts are installed as part of the Cluster Logging Operator installation.  The alerts depend upon the collection of metrics which are enabled during operator installation from the administration console by selecting the option  *Enable operator recommended cluster monitoring on this namespace*